tutorials/asr/Lhotse_Dataloading.ipynb

```diff
@@ -0,0 +1,459 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c14eb898-0f8e-4783-a70e-ab54b4de910d",
+   "metadata": {},
+   "source": [
+    "# Tutorial: NeMo & Lhotse Data Loading\n",
+    "\n",
+    "![image](https://raw.githubusercontent.com/lhotse-speech/lhotse/master/docs/logo.png)\n",
```

Comment:
logo shows up as very large when I view it here (https://github.com/NVIDIA/NeMo/blob/lhotse-dataloading-tutorial/tutorials/asr/Lhotse_Dataloading.ipynb), please make it smaller. I think you will have to use HTML tags